recce/tasks/query.py

```diff
@@ -233,5 +233,19 @@ class QueryDiffResultDiffer(TaskResultDiffer):
     def _check_result_changed_fn(self, result):
         base = result.get('base')
         current = result.get('current')
+        diff = result.get('diff')
 
-        return TaskResultDiffer.diff(base, current)
+        if diff is None:
+            return TaskResultDiffer.diff(base, current)
+        else:
+            base = dict(columns=[], data=[])
```

Comment:
Would it better to just check the row count of `diff` dataframe. If the size is not zero, there is diff?